src/ts/components/account/NetworkCard.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+interface INetworkCardProps {
+  imgPath: string
+  name: string
+  supported: boolean
+}
+
+interface INetworkCardState {
+  selected: boolean
+}
+
+export default class NetworkCard extends React.Component<INetworkCardProps, INetworkCardState> {
+
+  state = {
+    selected: false
+  }
+
+  render () {
+    return this.state.selected ? this.renderClicked() : this.renderDefault()
```

Comment:
so this is only UI. we don't store the network selection, right?

Reply:
I think in future we will store network selection from each account for organizing with use cases. Some might want an account to be used only on certain networks(e.g. an account for confidential transaction network such as zerochain or zcash parachain). I want to discuss more on this actually.